[Style Guide] Content Strategy (#9846)
cloudflare/cloudflare-docs: content/style-guide/content-strategy/documentation-content-strategy/content-types/support-content.md

---
pcx_content_type: concept
title: Support content

---

# Support content

## Purpose

The purpose of support content is to provide guidance for solving specific problems with Cloudflare.

## Tone

Guiding, straightforward, solution-oriented

## content_type

Use the metadata tag for the Content Type you create within the /support/ tile.</td>

### Required components

> Title<br/>Context

Reviewer: ```suggestion
Title

Context
```

### Optional components

> Steps/Tasks/Procedures<br/>Examples<br/>Next steps

Reviewer: ```suggestion
Steps/Tasks/Procedures

Examples

Next steps
```

## Structure

The structure of a page within the support tile will depend on the purpose that content serves.

## Boundaries

The general guidance is that anything related to a specific product (DNS, Security Center) or platform (Cloudflare One) belongs in a [product](https://developers.cloudflare.com/products/) tile. Anything that's specific to interacting with Cloudflare support or details niche troubleshooting steps belongs in the [support](https://developers.cloudflare.com/support/) tile.

Support content should not duplicate content that already exists in a product tile.

<table>
    <tr>
        <th style="width:50%">Support content examples</th>
        <th>Product content examples</th>
    </tr>
    <tr>
        <td>How to interact with Cloudflare support</td>

>         <td>Product documentation - feature info, functionality, configuration, known limitations</td>

Reviewer: ```suggestion
        <td>Product documentation - feature information, functionality, configuration, known limitations</td>
```

    </tr>
    <tr>
        <td>Niche troubleshooting information (multiple product configurations affecting each other)</td>
        <td>Troubleshooting a specific product or platform</td>
    </tr>
    <tr>

>         <td>Third-party integrations and partnerships that aren't specific to a product(like WordPress)</td>

Reviewer: ```suggestion
        <td>Third-party integrations and partnerships that are not specific to a product (like WordPress)</td>
```

        <td>Product-specific integration guides</td>
    </tr>
    <tr>
        <td>Runtime errors when integrating with external products</td>
        <td>Runtime errors specific to a product</td>
    </tr>
    <tr>
        <td>How Cloudflare charges for specific things (rates, usage)</td>

>         <td>Process documentation for how to set up billing information, etc - no pricing content</td>

Reviewer: ```suggestion
        <td>Process documentation for how to set up billing information, etc. - no pricing content</td>
```

    </tr>
</table>

## Additional information

> Migrating content from /support/ to other areas of dev docs
> When you migrate content from the Support tile to other, relevant product areas:

Reviewer: ```suggestion
Migrating content from /support/ to other areas of Developer Documentation

When you migrate content from the Support tile to other, relevant product areas:
```
